cymulate-web-application-firewall/info.json

```diff
@@ -0,0 +1,405 @@
+      "parameters": [
+        {
+          "title": "Site ID",
+          "name": "SiteID",
+          "type": "text",
+          "description": "Specify the site ID",
+          "tooltip": "Specify the site ID",
+          "required": true,
+          "visible": true,
+          "editable": true
+        },
+        {
+          "title": "Offset",
+          "name": "skip",
+          "type": "integer",
+          "description": "Specify the index of the first item to be returned by this operation. This parameter is useful if you want to get a subset of records, say records starting from the 10th index.",
+          "tooltip": "Specify the index of the first item to be returned by this operation. This parameter is useful if you want to get a subset of records, say records starting from the 10th index.",
+          "required": false,
+          "visible": true,
+          "editable": true
+        },
+        {
+          "title": "Limit",
+          "name": "limit",
+          "type": "integer",
+          "description": "Specify the maximum number of results that this operation should return. The minimum supported value is 1000 and the maximum supported value is 10000.",
+          "tooltip": "Specify the maximum number of results that this operation should return. The minimum supported value is 1000 and the maximum supported value is 10000.",
+          "required": false,
+          "visible": true,
+          "editable": true
+        }
+      ],
+      "output_schema": {}
+    },
+    {
+      "operation": "launch_waf_assessment",
+      "annotation": "launch_waf_assessment",
+      "title": "Launch WAF Assessment",
+      "description": "Launch a Web Application Firewall assessment based on the parameters that you have specified.",
+      "category": "investigation",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Sites",
+          "name": "sites",
+          "type": "text",
+          "description": "Specify the comma separated values of urls. eg: https://example1.com,https://example2.com",
+          "tooltip": "Specify the comma separated values of urls. eg: https://example1.com,https://example2.com",
+          "placeholder": "https://example1.com,https://example2.com",
+          "required": true,
+          "visible": true,
+          "editable": true
+        },
+        {
+          "title": "Template ID",
+          "name": "templateID",
+          "type": "text",
+          "description": "Specify the template ID.",
+          "tooltip": "Specify the template ID.",
+          "required": true,
+          "visible": true,
+          "editable": true
+        },
+        {
+          "title": "Schedule",
+          "name": "schedule",
+          "type": "datetime",
+          "description": "Specify the datetime of the schedule.",
+          "tooltip": "Specify the datetime of the schedule.",
+          "required": false,
+          "visible": true,
+          "editable": true
+        },
+        {
```

Comment:
Please check if this parameter can be changed to select , if not check if other options could be suggested

Reply:
this is uncertified connector and in the doc also any information is not provided for this parameter